infra/integration-image/Dockerfile.test

```diff
@@ -0,0 +1,37 @@
+# docker build -t vic-test -f Dockerfile.test .
+FROM golang:1.6.2
+
+RUN apt-get update
+RUN apt-get install -y jq time
+RUN curl -sSL https://get.docker.com/ | sh
+
+RUN apt-get install -y gcc python-dev python-setuptools python-pip libffi-dev libssl-dev
+RUN pip install pyasn1 gsutil --upgrade
+RUN pip install robotframework
+RUN pip install robotframework-sshlibrary
+
+# Libraries required to run the docker integration tests
```

Comment:
Putting the core Go dependencies in the image definition like this means that this isn't just a base image for running tests in, it's an image that captures the tests to be run at a point in time.
That actually has value of it's own (we can assert that we pass/fail tests as of a specific date) - however:
1. these aren't pulled from a tagged release so it's luck as to what we get when the image is built
2. we *want* early exposure to changes in these packages - we'll get some of it by our updates to vendor, but this is another vector
3. these packages are already in the repo vendor directory.